api/kyverno/v1/image_verification_types.go

```diff
@@ -254,6 +269,25 @@ type Attestation struct {
 	Conditions []AnyAllConditions `json:"conditions,omitempty" yaml:"conditions,omitempty"`
 }
 
+type ImageRegistryCredentials struct {
+	// Helpers specifies a list of OCI Registry names, whose authentication helpers are provided
+	// It can be of one of these values: AWS, ACR, GCP, GHCR
+	// +kubebuilder:validation:Optional
+	Helpers []ImageRegistryCredentialsHelpersType `json:"helpers,omitempty" yaml:"helpers,omitempty"`
+
+	// Secrets specifies a list of secrets that are provided for credentials
+	// Secrets must be in Kyverno namespace
+	// +kubebuilder:validation:Optional
+	Secrets []Secret `json:"secrets,omitempty" yaml:"secrets,omitempty"`
```

Comment:
@eddycharly Not any particular reason anymore, Initially I wanted to add support for secrets in different namespaces as well. But since secrets must be in Kyverno namespace, we can change it to []string now.

This implementation also matches with the schema mentioned in the issue:
```
imageRegistryCredentials
   helpers:
   - AWS
   - Azure
   secrets:
   - name: regcred
```

Reply:
fixed